presentation/matrix_view.py

```diff
@@ -214,29 +227,30 @@ def get_text(metric: MetricType, cell: MeshColumn) -> str:
 
     @staticmethod
     def make_hover_text(mesh: MeshResults) -> List[List[str]]:
-        text = []
+        matrix_hover_text: List[List[str]] = []
         for row in reversed(mesh.rows):
-            text_col = []
+            column_hover_text: List[str] = []
             for col in row.columns:
                 from_agent = mesh.agents.get_by_id(row.agent_id)
                 to_agent = mesh.agents.get_by_id(col.agent_id)
                 conn = mesh.connection(from_agent.id, to_agent.id)
-                latency_ms = conn.latency_millisec.value
-                jitter_ms = conn.jitter_millisec.value
-                loss = conn.packet_loss_percent.value
                 distance_km = calc_distance_in_kilometers(from_agent.coords, to_agent.coords)
-                text_col.append(
-                    f"{from_agent.alias} -> {to_agent.alias} <br>"
-                    + f"Distance: {distance_km:.0f} km<br>"
-                    + f"Latency: {latency_ms:.2f} ms <br>"
-                    + f"Jitter: {jitter_ms:.2f} ms <br>"
-                    + f"Loss: {loss:.1f}%"
-                )
-            text.append(text_col)
+                cell_hover_text = f"{from_agent.alias} -> {to_agent.alias} <br>" + f"Distance: {distance_km:.0f} km<br>"
```

Comment:
This code is obviously from the version before distance units were made configurable, right?

Reply:
Yes this is the first implementation of distance between agents.